cypress/page_object/task3/Header.js

```diff
@@ -0,0 +1,51 @@
+class Header {
```

Comment:
i think you need to separate functionality into several classes like SearchPanelForm, etc
you may still have them inside of this class though